dtests/src/resources/scripts/create_and_load_row_table.sql

```diff
@@ -0,0 +1,12 @@
+-- DROP TABLE IF ALREADY EXISTS --
+DROP TABLE IF EXISTS STAGING_AIRLINEREF;
+DROP TABLE IF EXISTS AIRLINEREF;
+
+----- CREATE TEMPORARY STAGING TABLE TO LOAD PARQUET FORMATTED DATA -----
+CREATE TABLE STAGING_AIRLINEREF
+    USING parquet OPTIONS(path '/home/swati/snappy-commons/build-artifacts/scala-2.10/snappy/quickstart/data/airportcodeParquetData');
```

Comment:
remove this file and use the already checked-in file. parquet file path can  be provided using as a parameter to script.

see below the syntax of the run command.

run -file=<path or URL> [-auth-provider=<name>] [-bind-address=<address>] [-client-bind-address=<address>] [-client-port=<port>] [-connection-url=<url>] [-encoding=<charset>] [-extra-conn-props=<properties>] [-help]
       [-ignore-errors] [-J-D<property=value>] [-locators=<addresses>] [-mcast-address=<address>] [-mcast-port=<port>] [-numTimesToRun=<numTimesToRun>] [-param:<param>] [-password[=<password>]] [-path=<path>] [-user=<username>]